msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/PermissionLevelManagerImpl.java

```diff
@@ -913,6 +921,7 @@ private PermissionsMask getDefaultReviewerPermissionsMask() {
 		  mask.put(PermissionLevel.READ, Boolean.valueOf(true));
 		  mask.put(PermissionLevel.MARK_AS_READ,Boolean.valueOf(true));
 		  mask.put(PermissionLevel.MODERATE_POSTINGS, Boolean.valueOf(false));
+		  mask.put(PermissionLevel.IDENTIFY_ANON_AUTHORS, Boolean.valueOf(false));
 		  mask.put(PermissionLevel.DELETE_OWN, Boolean.valueOf(false));
 		  mask.put(PermissionLevel.DELETE_ANY, Boolean.valueOf(false));
 		  mask.put(PermissionLevel.REVISE_OWN, Boolean.valueOf(false));
```

Comment:
This is where the default permissions are set; they differ for each role.
We could create a method like:
setupMask(Mask mask, boolean newForum, boolean newTopic, bolean...)
{
    mask.put(PermissionLevel.NEW_FORUM, newForum);
    mark.put(...
}
and invoke setupMask(mask, true, true, true, ...);
But then it'd be hard to read which boolean is going to which permission when we invoke it.
IMO it's better to lease all of these explicit.